src_py/apiServer/receiver.py

```diff
@@ -77,10 +85,8 @@ def post(self):
 
 #Listener Server list of resources: 
 api.add_resource(test, "/test")
-api.add_resource(train, "/train")
 api.add_resource(ack, "/ack")
 api.add_resource(shutdown, "/shutdown")
-api.add_resource(predict, "/predict")
-api.add_resource(testglobe, "/testglobe")
 api.add_resource(lossFunction, "/lossFunction")
+api.add_resource(predictRes, "/predictRes")
```

Comment:
If you can change predict with inference in all code (not urgent) 
@kapelnik @dordor7 @evgenyan95